### PR TITLE
Add blog index

### LINKED ISSUE #5: Page: /blog
Build out a blog list pages that lists all the blog posts

Samples:

https://turbo.build/blog
https://github.com/vercel/turbo/blob/main/docs/pages/blog.mdx

https://wagmi.sh/blog
https://github.com/wagmi-dev/wagmi/blob/main/docs/pages/blog.en-US.mdx

https://million.dev/blog
https://github.com/aidenybai/million/blob/main/website/pages/blog.mdx

### PR DESCRIPTION
**Pull request by carlbarrdahl**
Fixes #5

### CODE REVIEW

File: pages/blog.mdx [hunk 1]
```diff
@@ -1,3 +1,48 @@
...
+---
+description: The latest updates and releases from the team at Allo.
+type: posts
+---
 
-test
+import { getPagesUnderRoute } from "nextra/context";
+import Link from "next/link";
+
+export function BlogHeader() {
```
Comment (0xZakk) [inline pages/blog.mdx:9]: Could you pull these out into components in the `components/` directory? One per file, so a `components/BlogHeader.tsx` file and a `components/BlogIndex.tsx` file

**Review by 0xZakk — COMMENTED** [review]
@carlbarrdahl - one very small thing, otherwise this is fantastic. Thank you for the quick turn around!

File: pages/blog.mdx (hunk 1 above)
Comment (carlbarrdahl) [reply pages/blog.mdx]: Fixed!

**Review by 0xZakk — APPROVED** [review]
Thank you! This looks amazing!

**Pull request merged**